README.md

```diff
@@ -57,8 +57,8 @@ new Linnia(web3 [, options])
 1. `Object` - An instantiated web3 API object
 1. `Object` - (Optional) Constructor options
 
-- `linniaContractUpgradeHubAddress`: `String` - Address of the LinniaHub. If not specified, Linnia Javascript API will attempt to find the address of the deployed LinniaHub on the network defined in `web3`.
-- `linniaTokenContractAddress`:`String` - Address of the LinniaToken.  The Ropsten testnet Contract address is `0x4cdfbdec0aa003116bf030f249a8a7285cd6a184`
+- `hubAddress`: `String` - Address of the LinniaHub. If not specified, Linnia Javascript API will attempt to find the address of the deployed LinniaHub on the network defined in `web3`.
+- `tokenAddress`:`String` - Address of the LinniaToken.  The Ropsten testnet Contract address is `0x4cdfbdec0aa003116bf030f249a8a7285cd6a184`
```

Comment:
Same

Reply:
Also can remove the hardcoded address here

Reply:
ok